test/TestAssets/TestPackages/dotnet-portable/dotnet-portable.csproj

```diff
@@ -1,8 +1,15 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--<Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), testAsset.props))\testAsset.props" />-->
+
   <PropertyGroup>
     <!-- netcoreapp2.2 is the maximum TFM project tools support -->
     <TargetFramework>netcoreapp2.2</TargetFramework>
     <OutputType>Exe</OutputType>
+    <IsPackable>true</IsPackable>
```

Comment:
I'm surprised this worked before without `IsPackable`? Is this due to some logic where now that it's in a 'test' folder the SDK realizes its a test asset so tries not to make a nuget package out of it?

Reply:
You misunderstand. This is the current file in the SDK repo: https://github.com/dotnet/sdk/blob/main/src/Assets/TestPackages/dotnet-portable/dotnet-portable.csproj

Again, I didn't modify this file. It is the SDK's file merged into the existing Installer's file: https://github.com/dotnet/installer/blob/main/TestAssets/TestPackages/dotnet-portable/dotnet-portable.csproj

`test/TestAssets` **IS** the Installer repo's TestAssets folder. I'm merging the two so that all test assets are in `test/TestAssets`. When I finish the PR to make installer tests work, both will the SDK tests and Installer tests will use this same project file.

Reply:
Ah I see. That makes sense, because this PR seemed familiar with something you had done in the past. I didnt realize it was the installer repo's folder that was in this repo, thanks for elaborating :)